### PR TITLE
Added additional guidance for signing off git commits when contributing.

### PR DESCRIPTION
**Pull request by vdemeester**
From docker/docker#17565, @pmalmgren (Peter Malmgren <ptmalmgren@gmail.com>).

/ping @moxiegirl 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks! LGTM

File: docs/project/set-up-git.md [hunk 1]
```diff
@@ -92,8 +92,9 @@ You indicate your agreement by signing your `git` commits like this:
 
 To create a signature, you configure your username and email address in Git.
 You can set these globally or locally on just your `docker-fork` repository.
-You must sign with your real name. We don't accept anonymous contributions or
-contributions through pseudonyms.
+You must sign with your real name. You can sign your git commit automatically
+with `git commit -s`. We don't accept anonymous contributions or contributions
```
Comment (moxiegirl) [inline docs/project/set-up-git.md]: Docker does not...

@vdemeester I am trying to kill the :crown: in our docs (royal we).

Comment (thaJeztah) [inline docs/project/set-up-git.md]: Yup, I saw it, but it was the original text :innocent:

Comment (moxiegirl) [inline docs/project/set-up-git.md]: Well, I never claimed to be perfect just to be trying. :stuck_out_tongue:

Comment (vdemeester) [inline docs/project/set-up-git.md]: @moxiegirl ok, will take care of this :wink:

**Comment by thaJeztah** [comment]
thanks Vincent!

LGTM


ping @moxiegirl for the merge :-)

**Comment by moxiegirl** [comment]
@vdemeester thank you kind sir.

**Pull request merged**